web/src/pages/task/task-page.tsx

```diff
@@ -150,7 +150,7 @@ const TaskPage: FC = () => {
     };
 
     const handleSaveTaskSuccess = () => {
-        notifySuccess(<Text>Saved successfully</Text>);
+        notifySuccess(<Text>All unsaved changes were saved</Text>);
```

Comment:
Looks a bit repetitive, what do you think just about "All changes were saved"?

Reply:
Idk, this text was given to me by our QA.

Reply:
Changed